ckan/templates/home/index.html

```diff
@@ -8,7 +8,7 @@
 
 {% block content %}
   <div class="homepage layout-{{ homepage_style }}">
-    <div class="container">
+    <div id="content" class="container">
```

Comment:
Any reason why this was added here when there is already an `id="content"` in the page.html? https://github.com/ckan/ckan/blob/0e3f501e9fe36289cee4fc59647db80feb0e2f97/ckan/templates/page.html#L21

But this div doesn't seem to get rendered (at least not in 2.8).  If it's supposed to be here, maybe we should remove that other id. But would be nice to know why it wasn't working to start with.